src/cmd/CodeceptCommandBuilder.php

```diff
@@ -40,6 +43,11 @@ public function beSilent() {
         $this->_verbose = false;
     }
 
+    /** Disable verbose composer output. */
```

Comment:
Deze docstring klopt nu niet, maar kan misschien wel helemaal weg :)

Reply:
vergeten aan te passen. Is nu goed.